framework/wazuh/core/active_response.py

```diff
@@ -13,84 +13,221 @@
 from wazuh.core.wazuh_socket import create_wazuh_socket_message
 
 
-def create_message(command: str = '', custom: bool = False, arguments: list = None) -> str:
-    """Create the message that will be sent.
-
-    Parameters
-    ----------
-    command : str
-        Command running in the agent. If this value starts with !, then it refers to a script name instead of a command
-        name.
-    custom : bool
-        Whether the specified command is a custom command or not.
-    arguments : list
-        Command arguments.
-
-    Raises
-    ------
-    WazuhError(1650)
-        If the command is not specified.
-    WazuhError(1652)
-        If the command is not custom and the command is not one of the available commands.
+def get_commands() -> list:
+    """Get the available commands.
 
     Returns
     -------
-    str
-        Message that will be sent to the socket.
+    list
+        List with the available commands.
     """
-    if not command:
-        raise WazuhError(1650)
-
-    commands = get_commands()
-    if not custom and command not in commands:
-        raise WazuhError(1652)
-
-    msg_queue = "!{}".format(command) if custom else command
-    msg_queue += " " + " ".join(shell_escape(str(x)) for x in arguments) if arguments else " - -"
+    commands = list()
+    with open(common.AR_CONF) as f:
+        for line in f:
+            cmd = line.split(" - ")[0]
+            commands.append(cmd)
 
-    return msg_queue
+    return commands
 
 
-def create_json_message(command: str = '', arguments: list = None, alert: dict = None) -> str:
-    """Create the JSON message that will be sent. Function used when Wazuh agent version is >= 4.2.0.
+def shell_escape(command: str = '') -> str:
```

Comment:
Why do you need a default for the `command` parameter?

Reply:
Deleted the default value for the `command` parameter in commit https://github.com/wazuh/wazuh/commit/4d48ef998a04289f0d4dfa7f394d39a0d864fc3c.